pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -515,6 +540,7 @@ public static void addColumnMetadataInfo(PropertiesConfiguration properties, Str
     properties.setProperty(getKeyFor(column, HAS_DICTIONARY), String.valueOf(hasDictionary));
     properties.setProperty(getKeyFor(column, TEXT_INDEX_TYPE), textIndexType.name());
     properties.setProperty(getKeyFor(column, HAS_INVERTED_INDEX), String.valueOf(hasInvertedIndex));
+    properties.setProperty(getKeyFor(column, HAS_FST_INDEX), String.valueOf(hasFSTIndex));
```

Comment:
This is probably not needed right? In fact, may be should add a new enum to TEXT_INDEX_TYPE to something like LUCENE_FST. TEXT_INDEX_TYPE is already serialized in on-disk metadata so just extend it to capture FST

Reply:
Wouldn't this make only one of the indicies be available?